src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -50,6 +50,7 @@ public function getConfigTreeBuilder()
                     ->addDefaultsIfNotSet()
                     ->children()
                         ->booleanNode('enabled')->defaultTrue()->info('To enable or disable eager loading')->end()
+                        ->booleanNode('fetch_partial')->defaultFalse()->info('Fetch only partial data according to serialization groups')->end()
```

Comment:
I think we should add:

"If enabled, Doctrine ORM entities will **NOT** work as expected if any of the other fields are used."

:stuck_out_tongue_closed_eyes: